puppet/zulip/templates/nginx/zulip-enterprise.template.erb

```diff
@@ -15,6 +15,8 @@ server {
 include /etc/nginx/zulip-include/upstreams;
 include /etc/zulip/nginx_sharding_map.conf;
 
+proxy_cache_path /srv/zulip-uploaded-files-cache levels=1:2 keys_zone=uploads:1M;
```

Comment:
What does the `1M` here do? How does this configuration control how much disk we use? 

(I'm pondering if this is something that should involve a comment, a parameter, or neither).

Reply:
Yeah, that's the cache size.  I think we'll want a knob in `zulip.conf` for it.